test/h5test.c

```diff
@@ -386,20 +386,30 @@ h5_reset(void)
 /*
  * Performs test framework initialization
  */
-void
+herr_t
 h5_test_init(void)
 {
+    int ret = 0;
```

Comment:
Match `herr_t` and use `SUCCEED` / `FAIL` below instead of -1 in case `herr_t` changes in the future.